solutions/usaco-807.mdx

```diff
@@ -74,4 +74,38 @@ public class Teleportation {
 	
 </JavaSection>
 
+<CPPSection>
+
+```cpp
+#include <iostream>
+#include <stdio.h>
+#include <stdlib.h>
+using namespace std;
+
+int main(void) {
```

Comment:
this isn't c
```suggestion
int main() {
```

Reply:
ah sorry about that. I usually code a lot in c.